icons/moq.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Moq</title><path d="M17.544 9.062a.855.855 0 0 0-.876.863V11.7c0 .42.312.863.876.863s.875-.443.875-.863V9.925a.854.854 0 0 0-.875-.863Zm-5.544 0a.855.855 0 0 0-.876.863V11.7c0 .42.313.863.876.863s.876-.443.876-.863V9.925A.855.855 0 0 0 12 9.062ZM12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12S18.627 0 12 0ZM8.614 13.32c0 .167-.095.263-.263.263h-.636c-.168 0-.264-.096-.264-.264V9.241c0-.132-.096-.216-.24-.216s-.24.084-.24.216v4.078c0 .168-.095.264-.263.264h-.504c-.168 0-.264-.096-.264-.264V9.241c0-.132-.096-.216-.24-.216s-.24.084-.24.216v4.078c0 .168-.096.264-.264.264h-.635c-.168 0-.264-.096-.264-.264V9.241c0-.767.432-1.32 1.33-1.32.266 0 .541.098.757.241a.15.15 0 0 0 .143 0c.216-.143.493-.24.756-.24.9 0 1.331.552 1.331 1.32v4.077Zm5.545-1.62c0 1.068-.864 2.003-2.159 2.003-1.296 0-2.16-.935-2.16-2.003V9.925c0-1.068.864-2.003 2.16-2.003 1.295 0 2.16.935 2.16 2.003V11.7Zm5.544 0c0 .804-.54 1.62-1.367 2.003-.072.036-.156.12-.156.228v.12c0 .504.383.78.588.84.168.047.263.095.263.263v.672c0 .18-.095.252-.263.252-.888-.024-1.86-.912-1.86-2.027v-.12c0-.108-.084-.192-.156-.228-.827-.384-1.367-1.2-1.367-2.003V9.925c0-1.068.864-2.003 2.159-2.003s2.16.935 2.16 2.003V11.7Z"/></svg>
```

Comment:
Hello @ripzerotwo, on comparing with the original, it is not matching up.
![image](https://github.com/simple-icons/simple-icons/assets/35000807/26d91820-19ab-4e9c-98fd-915e942a8a04)